src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -4176,6 +4164,42 @@ ApplicationManagerImpl::SupportedSDLVersion() const {
       get_settings().max_supported_protocol_version());
 }
 
+void ApplicationManagerImpl::AddAppToRegisteredAppList(
+    ApplicationSharedPtr application) {
+  LOG4CXX_AUTO_TRACE(logger_);
+  DCHECK_OR_RETURN_VOID(application);
+  sync_primitives::AutoLock lock(applications_list_lock_ptr_);
+
+  // Add application to registered app list and set appropriate mark.
+  application->MarkRegistered();
+  applications_.insert(application);
+  LOG4CXX_DEBUG(
+      logger_,
+      "App with app_id: " << application->app_id()
+                          << " has been added to registered applications list");
+  apps_size_ = static_cast<uint32_t>(applications_.size());
+}
+
+void ApplicationManagerImpl::AddExtensionsToApp(
+    ApplicationSharedPtr application) {
+  LOG4CXX_AUTO_TRACE(logger_);
+  DCHECK_OR_RETURN_VOID(application);
+  sync_primitives::AutoLock lock(applications_list_lock_ptr_);
+
+  std::function<void(plugin_manager::RPCPlugin&)> on_app_registered =
```

Comment:
@IGapchuk auto ?

Reply:
@AKalinich-Luxoft Has been changed in e792e3aa3409c421efb5f81bb14a5e7013b9ed0b